api/schema.go

```diff
@@ -2,48 +2,59 @@ package api
 
 import (
 	"encoding/xml"
-	"github.com/Albert221/medicinal-products-registry-api/data"
-	"github.com/graph-gophers/graphql-go"
 	"io/ioutil"
-	"log"
+
+	"github.com/Albert221/medicinal-products-registry-api/data"
+	graphql "github.com/graph-gophers/graphql-go"
+)
+
+const (
+	medProductsDataSetPath string = "downloads/dataset.xml"
+	graphqlSchemaPath      string = "schema.graphql"
 )
 
+// Schema represents API Schema
 type Schema struct {
 	medicalProducts *data.MedicalProducts
 }
 
-func NewSchema(updateChan chan bool) *Schema {
-	schema := &Schema{getMedicalProducts()}
-
-	go func() {
-		for {
-			<-updateChan
-			schema.medicalProducts = getMedicalProducts()
-		}
-	}()
-
-	return schema
+// NewSchema return a pointer to API Schema
+func NewSchema() (*Schema, error) {
+	pp, err := getMedicalProducts()
```

Comment:
What's the double *p* from? :)

Reply:
Double p means plural products contraction. But I renamed it back to products to make it more clear. Sorry for that